# - without installerType set jpackage tries to create rpm and deb
#   we don't need any of them, but I did not find a way to disable
#   both. so we leave rpm which is available in conda (spkg isn't)
diff -ruN mzmine3-3.6.0-org/build.gradle mzmine3-3.6.0/build.gradle
--- mzmine3-3.6.0-org/build.gradle	2023-06-21 14:39:23.000000000 +0200
+++ mzmine3-3.6.0/build.gradle	2023-06-22 17:53:22.239977816 +0200
@@ -392,7 +392,7 @@
         if (OperatingSystem.current().isLinux()) {
             // leave installer type empty to generate all that match
             // for rpm, rpmbuild and rpm tools need to be installed, installerType needs to be set to "rpm"
-            // installerType = "deb"
+            installerType = "rpm"
             // additional options for jpackage
             imageOptions = ["--icon", "src/main/resources/MZmineIcon.png"]
             installerOptions = ["--linux-shortcut",
